src/components/CheckBox/__test__/index.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
now that you've made `checkbox` a reusable component you should write its tests independent of its usage so there shouldn't be any sign of todos in this file.